bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv -types',
```

Comment:
no, I believe executable is `tomlv`. If you need to append that argument, do that in create_arguments method.